ranger/core/main.py

```diff
@@ -119,6 +119,9 @@ def main(
         FileManagerAware.fm_set(fm)
         load_settings(fm, args.clean)
 
+        if args.choosedir:
+            SettingsAware.settings.global_inode_type_filter = 'd'
```

Comment:
AFAIK (and can find via google) the accepted way to name constants in python is to use all caps (such as `FILTER_DIR`). But I am getting pylint errors complaining that this is an 'invalid variable name'?